corobo/docker-compose.yml

```diff
@@ -4,3 +4,8 @@ services:
     image: meetmangukiya/corobo
     restart: always
     env_file: .secrets
+  answers:
+    image: meetmangukiya/corobo-answers
+    restart: always
+    expose:
+      - "8000"
```

Comment:
No, don't expose the port. Expose it inside the Dockerfile of `meetmangukiya/corobo-answers` and link the container to corobo.